ui/app/templates/partials/replication/enable.hbs

```diff
@@ -185,7 +185,7 @@
             {{#if showExplanation}}
               <p>
                 When running as a DR secondary Vault is read only.
```

Comment:
Less sure about this one...
```suggestion
                When running as a DR Secondary Vault is read only.
```

Reply:
Definitely a gray area on these. Depends on if you read it as "DR Secondary" (which could be a feature), or if you read "secondary" as separate. Capitalized it for now anyway